src/popup/router/pages/Networks.vue

```diff
@@ -59,6 +59,18 @@
         v-model="network.compilerUrl"
         data-cy="compiler"
       />
+      <div class="text-left expand" @click="expanded = !expanded">
+        <img :class="expanded ? 'open' : ''" src="../../../icons/carret-down.svg" />
+        <span>{{
+          expanded ? $t('pages.network.hideTippingConfig') : $t('pages.network.showTippingConfig')
+        }}</span>
+      </div>
+      <Input
+        v-show="expanded"
```

Comment:
why don't use `v-if`?

Reply:
It is part of the form, the value would be taken into consideration no matter if it is expanded or not - this is the reasoning behind v-show.

Reply:
The absence of Input won't break `backendUrl` value. `v-if` is a bit more performant and clear, so use it unless you have a strong reason to use `v-show`.